lib-py/casmutils/xtal/coordinate.py

```diff
@@ -29,7 +51,43 @@ def frac(self, lat):
         np.array
 
         """
-        return self._frac_const(lat)
+        return self._pybind_value._frac_const(lat)
+
+    @classmethod
+    def _from_pybind(cls, py_bind_value):
```

Comment:
We don't have to address it for this pull request, but is there a clever way to turn the `pybind` related stuff into a base class that we can reuse? Maybe it could include members like `_pybind_value`, `_pybind_type`, and `_from_pybind`? Something to think about.